src/libs/ajax.js

```diff
@@ -1226,6 +1226,27 @@ const Surveys = signal => ({
   })
 })
 
+const WorkspaceDataService = signal => ({
+  getSchema: async instanceId => {
+    const res = await fetchWDS(`${instanceId}/types/v0.2`, _.merge(authOpts(), { signal }))
+    return res.json()
+  },
```

Comment:
Alternatively, these can be written more concisely like:

```suggestion
  getSchema: instanceId => fetchWDS(`${instanceId}/types/v0.2`, _.merge(authOpts(), { signal })).then(r => r.json()),
```

Reply:
probably a preference thing... I prefer the async await style here.... and I like having the const on it's own line that is easier to set a breakpoint on before the return.... (in the event of a local or prod issue that needs debugging)

Reply:
Agreed it's a preference thing, I won't request changes for one way or the other.